### ISSUE TITLE
Create a prepanel function for freqpolygon

### ISSUE THREAD

**Issue by rpruim** [ProjectMOSAIC/mosaic#486]
Should have done this long ago, but my work on ASH plots (#485) got me to figuring this out.

**Comment by rpruim** [comment]
At long last I've dealt with this.  It is a bit of a pain because the right wrapper to use is `densityplot()`, but it does special things with a few arguments, including `width`.  Once I figured out that this was being stored in `darg` and not passed along downstream I got this to work by creating a new prepanel function and using `darg` where necessary to recover `width`.

More testing would be good, but things seem to be behaving properly now.  `ylim` should be set to a value that contains all of the plotted points by default.

```r
freqpolygon( ~i2 | substance + sex, groups = homeless , data = HELPrct, width=10, auto.key=TRUE)
```

![image](https://cloud.githubusercontent.com/assets/722231/8370339/8bc51da6-1c1a-11e5-8501-de307349a7cb.png)
